app/src/main/java/org/schabi/newpipe/about/AboutActivity.java

```diff
@@ -146,16 +146,20 @@ public View onCreateView(@NonNull final LayoutInflater inflater, final ViewGroup
             aboutBinding.appVersion.setText(BuildConfig.VERSION_NAME);
 
             aboutBinding.githubLink.setOnClickListener(nv ->
-                    openUrlInBrowser(context, context.getString(R.string.github_url)));
+                    openUrlInBrowser(context, context.getString(R.string.github_url),
+                            false));
 
             aboutBinding.donationLink.setOnClickListener(v ->
-                    openUrlInBrowser(context, context.getString(R.string.donation_url)));
+                    openUrlInBrowser(context, context.getString(R.string.donation_url),
+                            false));
 
             aboutBinding.websiteLink.setOnClickListener(nv ->
-                    openUrlInBrowser(context, context.getString(R.string.website_url)));
+                    openUrlInBrowser(context, context.getString(R.string.website_url),
+                            false));
```

Comment:
Could you move the `false`s that would fit into the 100 character limit inline? Also above